packages/@o3r/apis-manager/schematics/helpers/update-api-deps/index.ts

```diff
@@ -107,7 +107,7 @@ export function apiManagerFactory(): ApiManager {
     tsconfigObj.compilerOptions.lib.push('dom');
     tsconfigObj.compilerOptions.lib = tsconfigObj.compilerOptions.lib.reduce((acc: string[], lib: string) => acc.indexOf(lib) >= 0 ? acc : [...acc, lib], []);
 
-    tree.overwrite(tsconfig, commentJson.stringify(tsconfigObj, null, 2));
+    tree.overwrite(tsconfig, JSON.stringify(tsconfigObj, null, 2));
```

Comment:
By doing this we loose the comment in this file no ?

Reply:
Yes we do, There is not way in Ts API to write tsconfig without loosing the comments :S